src/main/java/fi/metatavu/kuntaapi/server/index/IndexableService.java

```diff
@@ -0,0 +1,335 @@
+package fi.metatavu.kuntaapi.server.index;
+
+import java.util.Collections;
+import java.util.List;
+
+public class IndexableService implements Indexable {
+
+  @Field(index = "not_analyzed", store = true, type = "long")
+  private Long orderIndex;
+
+  @Field(index = "not_analyzed", store = true)
+  private String serviceId;
+
+  @Field(index = "not_analyzed", store = true)
+  private List<String> organizationIds;
+  
+  @Field (analyzer = "finnish")
+  private String nameFi;
+
+  @Field (analyzer = "swedish")
+  private String nameSv;
+
+  @Field (analyzer = "english")
+  private String nameEn;
+
+  @Field (analyzer = "finnish")
+  private String alternativeNameFi;
+
+  @Field (analyzer = "swedish")
+  private String alternativeNameSv;
+
+  @Field (analyzer = "english")
+  private String alternativeNameEn;
+  
+  @Field (analyzer = "finnish")
+  private String shortDescriptionFi;
+
+  @Field (analyzer = "swedish")
+  private String shortDescriptionSv;
+
+  @Field (analyzer = "english")
+  private String shortDescriptionEn;
+  
+  @Field (analyzer = "finnish")
+  private String userInstructionFi;
+
+  @Field (analyzer = "swedish")
+  private String userInstructionSv;
+
+  @Field (analyzer = "english")
+  private String userInstructionEn;
+  
+  @Field (analyzer = "finnish")
+  private String descriptionFi;
+
+  @Field (analyzer = "swedish")
+  private String descriptionSv;
+
+  @Field (analyzer = "english")
+  private String descriptionEn;
+
+  @Field (analyzer = "finnish")
+  private List<String> keywordsFi;
+
+  @Field (analyzer = "swedish")
+  private List<String> keywordsSv;
+
+  @Field (analyzer = "english")
+  private List<String> keywordsEn;
+
+  @Field(index = "not_analyzed", store = true)
+  private List<String> electronicServiceChannelIds;
+
+  @Field(index = "not_analyzed", store = true)
+  private List<String> phoneServiceChannelIds;
+
+  @Field(index = "not_analyzed", store = true)
+  private List<String> serviceLocationServiceChannelIds;
+
+  @Field(index = "not_analyzed", store = true)
+  private List<String> printableFormServiceChannelIds;
+
+  @Field(index = "not_analyzed", store = true)
+  private List<String> webPageServiceChannelIds;
+
+  @Override
+  public String getId() {
+    return getServiceId();
+  }
+
+  @Override
+  public String getType() {
+    return "service";
+  }
+
+  @Override
+  public Long getOrderIndex() {
+    return orderIndex;
+  }
+
+  public void setOrderIndex(Long orderIndex) {
+    this.orderIndex = orderIndex;
+  }
+
+  public String getServiceId() {
+    return serviceId;
+  }
+
+  public void setServiceId(String serviceId) {
+    this.serviceId = serviceId;
+  }
+
+  public List<String> getOrganizationIds() {
+    return organizationIds;
+  }
+
+  public void setOrganizationIds(List<String> organizationIds) {
+    this.organizationIds = organizationIds;
+  }
+
+  @Deprecated
+  public String getName() {
+    return null;
+  }
+
+  @Deprecated
+  public String getAlternativeName() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)